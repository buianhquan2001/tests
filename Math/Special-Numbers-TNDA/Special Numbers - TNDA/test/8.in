46
2608 721
1847 4849
937 3286
1139 2375
4217 4814
1965 2851
780 933
2722 4783
4498 1517
3762 900
3579 3886
4215 4659
2237 4725
2386 4175
3341 1884
3358 3378
1081 2249
1438 606
2729 3850
2521 4892
2129 2741
4115 3129
4317 1864
3168 679
1482 4978
1003 683
2540 2025
2056 2590
692 4875
945 1400
2927 4672
769 1098
1420 4287
2237 1567
4254 2399
3811 2761
4912 2500
4793 3647
929 4199
1119 2209
3156 1434
4956 1359
2543 2841
1792 3262
1765 892
2718 4427
