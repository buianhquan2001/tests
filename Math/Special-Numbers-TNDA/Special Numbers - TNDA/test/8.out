162319308
418567528
541753429
523694684
647576913
61727898
356396342
653299508
24167462
573197298
970394054
825145717
981470035
603696048
917155635
787013059
820550059
171487262
814411445
957018022
11933299
834005519
874276230
652184764
111701132
524148744
821206820
91091003
21626692
182701325
471918649
535045280
13787690
974664633
758390842
457485365
881505902
767615361
813278116
233994100
438396834
636489285
483800462
69667922
332404014
192579882
